context/app/static/js/pages/Workspaces/Workspaces.jsx

```diff
@@ -6,6 +6,33 @@ import Description from 'js/shared-styles/sections/Description';
 import { LightBlueLink } from 'js/shared-styles/Links';
 import WorkspacesList from 'js/components/workspaces/WorkspacesList';
 
+function AuthenticatedWorkspaces() {
```

Comment:
```suggestion
function AuthenticatedWorkspacesDescription() {
```
`AuthenticatedWorkspaces` gives a sense that the component is more than a simple description.

Reply:
I changed the name, but then remembered that it does contain `<WorkspacesList />`... So might change back.